library/ix-dev/community/frigate/upgrade_strategy

```diff
@@ -7,6 +7,7 @@ from catalog_update.upgrade_strategy import semantic_versioning
 
 
 RE_STABLE_VERSION_BASE = r'\d+\.\d+\.\d+'
+RE_BETA_VERSION_BASE = r'\d+\.\d+\.\d+-beta\d+'
```

Comment:
The `-beta` will have to be removed too when getting the version of the semantic versioning.
As it wont be sortable